pkg/webui/locales/ja.json

```diff
@@ -2415,6 +2415,7 @@
   "error:pkg/networkserver:schedule": "全てのスケジュールされたダウンリンクが失敗に終わりました",
   "error:pkg/networkserver:transmission": "ダウンリンク伝送が結果 `{result}` で失敗しました",
   "error:pkg/networkserver:unknown_mac_state": "不明なMAC状態",
+  "error:pkg/networkserver:unknown_mac_version": "",
```

Comment:
Do we need to add the translation here or can it be left blank?

Reply:
Honestly I don't know what the convention is here. @kschiffer are we translating this in ChatGPT or do we have this file translated once in a while by our Japanese partners?

Reply:
Yes, we do this periodically and leave this empty until then.